rero_ils/config.py

```diff
@@ -83,6 +83,10 @@ def _(x):
     return x
 
 
+# Cors
+REST_ENABLE_CORS = True
```

Comment:
Why did you this? It has no mention in the commit message.

Reply:
it's a mistake